docs/topics/mta-web-config-jira-connection.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-jira-connection_{context}"]
+= Configuring Jira connection
+
+To be able to create Jira issues from within the {ProductName} ({ProductShortName}) {WebName} and to track application migrations, you need to create a Jira instance in {ProductShortName} and establish a connection to that instance.
+
```

Comment:
Consider "To create Jira issues...migrations, you first need to ..." 

If a user wants to create a Jira connection, would they know this section is for them? Perhaps change the title to 
"Creating and configuring Jira connections" or "...a Jira connection".